### ISSUE TITLE
Entertained the idea of integrating eslint for roku projects?

### ISSUE THREAD

**Issue by jfrux** [rokucommunity/brighterscript#38]
Been toying around with `eslint-plugin-roku` project.

![image](https://user-images.githubusercontent.com/171077/74344771-152fed00-4d7b-11ea-84be-b1b447dde2a8.png)

Is there something similar to this but uses your warnings / error parser and suggests issues in line in `vscode`?

**Comment by TwitchBronBron** [comment]
No, there's nothing like this yet. It's on the roadmap, but I just can't justify spending time on a linter first before we have fully functional parsing. :D 

The AST we generate in BrighterScript is pretty non-standard, but there might be a way to translate it into an eslint-compatible format. 

i've also been toying with just building the linter into the parser itself, as warnings. That would improve performance and centralize the tools into a single executable. 

Not sure how I feel about that yet, but I'm open to discussion.
